docker/requirements.txt

```diff
@@ -1,30 +1,29 @@
-Flask_Sockets
-PyGithub
-aiodns
-aiohttp
-aiohttp_jinja2
-aiomysql
-asyncinit
-authlib
-cerberus
-decorator
-flake8
-flask
-flask-cors
-google-cloud-storage
-humanize
-jinja2
-kubernetes
-libsass
-pip
-pylint
-pymysql
-pytest
-requests
-urllib3<1.25,>=1.21.1
-uvloop>=0.12
-werkzeug
-parsimonious
-kubernetes_asyncio
+aiodns==2.0.0
+aiohttp==3.5.4
+aiohttp-jinja2==1.1.1
+aiomysql==0.0.20
+asyncinit==0.2.4
+Authlib==0.11
+Cerberus==1.3.1
+decorator==4.4.0
+flake8==3.7.7
+Flask==1.0.3
+Flask-Cors==3.0.8
+Flask-Sockets==0.2.1
```

Comment:
so weird that there are multiple ways to represent these ("_","-"), but checked, it installs fine.

Reply:
Yep, I was surprised by that, too.  `pip freeze` "outputs in requrements.txt format", so I figured that should be the canonical style.